### ISSUE TITLE
Re-add support for the Mueller kpoint scheme

### ISSUE THREAD

**Issue by arosen93** [materialsproject/pymatgen#2979]
In older versions of Pymatgen, there was a pymatgen.ext.jhu utility that would allow for generation of kpoints using the scheme from Mueller but it required access to a server (which is maybe why it was removed?).

As shown here 
https://gitlab.com/muellergroup/k-pointGridGenerator there is a way to generate these kpoints without the server (but with an optional dependency). It would be a nice feature to add one day. Maybe if I discover some free time.......... logging the idea here anyway.

**Comment by shyuep** [comment]
Yup it was removed because the server was far too fragile. Would be happy to have it back as an optional dependency.
